src/validations/vulnerable-dependencies.js

```diff
@@ -36,11 +36,23 @@ module.exports = {
                     .then((result) => {
                         if (vulnerabilitiesFoundIn(result)) {
                             const errs = new Set();
-                            result.actions.forEach((action) => {
-                                action.resolves.forEach((vulnerability) => {
-                                    errs.add(summaryOf(vulnerability));
-                                });
-                            });
+
+                            Object.keys(result.vulnerabilities).forEach(
+                                (vulnerability) => {
+                                    result.vulnerabilities[
+                                        vulnerability
+                                    ].nodes.forEach((path) => {
```

Comment:
We don't need to create separate lines for arrays or objects with lengths less than 3. If you want you can move `forEach` to a separate line
```suggestion
                                    result.vulnerabilities[vulnerability].nodes
                                        .forEach((path) => {
```

or left it in one line 
```suggestion
                                    result.vulnerabilities[vulnerability].nodes.forEach((path) => {
```